src/Components/Admin/EditUserForm.js

```diff
@@ -53,45 +29,27 @@ function EditUserForm({ dispatch, formState }) {
     [dispatch]
   );
 
-  const dispatchProgramsChange = useCallback(
-    (event) => {
-      dispatch({
-        type: EDIT_PROGRAMS,
-        program: event.target.name
-      });
-    },
-    [dispatch]
-  );
-
   return (
     <div>
-      <QuestionWrapper>
-        <StyledLabel htmlFor="name-Input">Name</StyledLabel>
-        <OutlinedInput
-          fullWidth
-          id="name-input"
-          onChange={dispatchNameChange}
-          value={formState.name}
-        />
-      </QuestionWrapper>
-      <QuestionWrapper>
-        <StyledLabel htmlFor="preferred-name-input">
-          Preferred Name (optional)
-        </StyledLabel>
-        <OutlinedInput
-          fullWidth
-          id="preferred-name-input"
-          onChange={dispatchPreferredNameChange}
-          value={formState.preferredName}
-        />
-      </QuestionWrapper>
+      {!newUser && (
+        <QuestionWrapper>
+          <StyledLabel htmlFor="name-Input">Name</StyledLabel>
+          <OutlinedInput
+            fullWidth
+            id="name-input"
+            value={formState.name}
+            disabled
```

Comment:
Admins are no longer able to change a user's name, so this is really just here to display the name in the dialog. This should be updated to reflect the designs later, but I won't do this now.